### PR TITLE
Update Lambda Step Notebook with content table and few updates

### PR DESCRIPTION
**Pull request by shreyapandit**
*Issue #, if available:*

*Description of changes:*
Update Lambda Step Pipelines Notebook with content table and few updates

*Testing done:*

Ran end to end on notebook Instance

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [ ] I have read the [CONTRIBUTING](https://github.com/aws/amazon-sagemaker-examples/blob/master/CONTRIBUTING.md) doc and adhered to the example notebook best practices
- [ ] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-sagemaker-examples/blob/master/README.md)
- [ ] I have tested my notebook(s) and ensured it runs end-to-end
- [ ] I have linted my notebook(s) and code using `tox -e black-format,black-nb-format`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: sagemaker-pipelines/tabular/lambda-step/sagemaker-pipelines-lambda-step.ipynb [hunk 1]
```diff
@@ -4,16 +4,16 @@
    "cell_type": "markdown",
```
Comment (atqy) [inline sagemaker-pipelines/tabular/lambda-step/sagemaker-pipelines-lambda-step.ipynb:4]: <p>The title of the notebook should be H1 style</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3298/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (atqy) [inline sagemaker-pipelines/tabular/lambda-step/sagemaker-pipelines-lambda-step.ipynb:4]: <p>I think we have to add the runtime too for golden notebooks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3298/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (atqy) [inline sagemaker-pipelines/tabular/lambda-step/sagemaker-pipelines-lambda-step.ipynb:4]: <p>Just a typo, missing the word 'effect' (i.e. 'acheive the same effect')</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3298/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (atqy) [inline sagemaker-pipelines/tabular/lambda-step/sagemaker-pipelines-lambda-step.ipynb:4]: <p>Just a typo: "s ession" -&gt; "session"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3298/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (atqy) [inline sagemaker-pipelines/tabular/lambda-step/sagemaker-pipelines-lambda-step.ipynb:4]: <p>No need for 'the': Setting up the Lambda -&gt; Setting up Lambda</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3298/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (atqy) [inline sagemaker-pipelines/tabular/lambda-step/sagemaker-pipelines-lambda-step.ipynb:4]: <p>Present tense must be used: "an IAM role that will allow it" -&gt; "an IAM role that allows it"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3298/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Comment (shreyapandit) [reply sagemaker-pipelines/tabular/lambda-step/sagemaker-pipelines-lambda-step.ipynb:4]: It is H1, curious where you see otherwise

**Pull request merged**